apps/designer/app/shared/db/project.server.ts

```diff
@@ -6,41 +6,55 @@ import {
   prisma,
   Prisma,
   Project as BaseProject,
+  Asset,
 } from "@webstudio-is/prisma-client";
 import * as db from ".";
+import { getAssetPath } from "@webstudio-is/asset-uploader/src/helpers/get-asset-path";
 
 const TreeHistorySchema = z.array(z.string());
 
 export type Project = Omit<BaseProject, "prodTreeIdHistory"> & {
   prodTreeIdHistory: z.infer<typeof TreeHistorySchema>;
 };
 
-const parseProject = (project: BaseProject | null): Project | null => {
+const parseProject = (
+  project: BaseProject | null
+): Project | Omit<BaseProject, "assets"> | null => {
   if (project === null) return null;
   const prodTreeIdHistory = JSON.parse(project.prodTreeIdHistory);
   TreeHistorySchema.parse(prodTreeIdHistory);
+  const newAssets =
+    "assets" in project
+      ? project.assets.map((asset) => ({
+          ...asset,
+          path: getAssetPath(asset),
+        }))
+      : {};
   return {
     ...project,
+    ...newAssets,
```

Comment:
shouldnt this be something like this?
```suggestion
    assets: newAssets,
```

Reply:
Or rather actually - you probably have meant to create `{ assets: projects.assets.map(() => {...}) }` a couple of lines ago

Reply:
They are both the same idea, just committed your first one :)

Reply:
In here there is a subtle difference and that's also why I've incidentally suggested that first patch at first 😅 It's on the verge of a typo. The problem is that with this patch your `newAssets` are still either an array or an object, the other suggested patch ensures that we spread an object with an optional `assets` property.